### DISCUSSION TITLE
How to get icons approved?

### DISCUSSION THREAD

**Discussion by alexfazio** [elrumo/macOS_Big_Sur_icons_replacements#1251] [Help with the website] [answered]
Is there any way to speed up the approval process? I like to contribute but the idea of my work being stuck on a limbo makes me hesitant.

**Comment by cake03** [comment, marked as answer]
Sorry, but right now there is no other way for icons to get approved than by elrumo.

**Reply by elrumo** [reply]
The approval process should be a lot faster now. If an icon doesn't get approved in a few days then it's safe to assume that it didn't meet the requirements.
